docs/concepts/bqm.rst

```diff
@@ -0,0 +1,52 @@
+.. _bqm_sdk:
+
+=======================
+Binary Quadratic Models 
+=======================
+
+The binary quadratic model (BQM) class contains
+Ising and quadratic unconstrained binary optimization (QUBO) models
+used by samplers such as the D-Wave system.
```

Comment:
I think we should also put the BQM equation here

Reply:
![image](https://user-images.githubusercontent.com/34041130/76016073-09929a80-5ed1-11ea-842a-f023a3aecb7c.png)